Support privilege changes (initially only on tables)

alvarogzp opened the pull request:
Add `privileges` drops and creations to `Migration.add_all_changes()`.
Updated `everything` test to check grant/revoke statements are generated as expected.

Notes:
- I have also included #39's changes here so that we can know if tests are passing (at least on Python 3.6).
- This pull request depend on https://github.com/djrobstep/schemainspect/pull/9 (and a new version of schemainspect library to be released)

alvarogzp commented:
Ok, tests are failing because changes are not yet on `schemainspect` library.

djrobstep commented:
So I've incorporated your changes into `schemainspect` and released a new version, so the tests now pass, yay.

Been thinking about the design of this and figure we should put this functionality behind a command flag (`--with-privileges` or something), as often when comparing databases, you don't necessarily care about privileges. What do you think? Would you be happy to add this?

alvarogzp commented:
I have added the `--with-privileges` flag to migra.

I have implemented it as an argument to `add_all_changes()` instead of a separate method because revoke statements need to be added before drop statements, and grant statements need to be added after create statements. As the ordering logic is in that method, I thought it was better to keep privileges there too.

I have also extended `test_migra` to support the new flag, and created a new `privileges` fixture to explicitly test them.

djrobstep commented:
Code looked great. Merged! Will do a new release shortly.

djrobstep commented:
I've now done a new release incorporating your changes. Thanks once again for this, this was the first truly feature-sized contribution I've received on this project, so I was especially delighted to receive it!

alvarogzp commented:
Thank you so much for being so open to accept new features. It's nice to work with owners like you!
